.env.example

```
@@ -0,0 +1,5 @@
+SECRET_KEY=change-me
+ADMIN_USERNAME=admin
+# 使用 `python -c "from app.core.security import hash_password; print(hash_password('your-password'))"` 生成
+ADMIN_PASSWORD_HASH=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD_HASH key should go before the ADMIN_USERNAME key

(UnorderedKey)